docs/source/commandline/commandline-all.rst

```diff
@@ -496,137 +680,373 @@ Core rotation options
 |
 
 
-Core artwork options
+Core Video Options
+------------------
+
+.. _mame-commandline-video:
+
+**-video** *<bgfx|gdi|d3d|opengl|soft|none>*
```

Comment:
There's also an `accel` option that uses SDL2 2D acceleration if available.

Reply:
Fixed on my side.